src/single-spa.ts

```diff
@@ -0,0 +1,56 @@
+export { start } from "./start.js";
+export { ensureJQuerySupport } from "./jquery-support.js";
+export {
+  setBootstrapMaxTime,
+  setMountMaxTime,
+  setUnmountMaxTime,
+  setUnloadMaxTime,
+} from "./applications/timeouts.js";
+export {
+  registerApplication,
+  unregisterApplication,
+  getMountedApps,
+  getAppStatus,
+  unloadApplication,
+  checkActivityFunctions,
+  getAppNames,
+  pathToActiveWhen,
+} from "./applications/apps.js";
+export {
+  navigateToUrl,
+  patchHistoryApi,
+} from "./navigation/navigation-events.js";
+export { triggerAppChange } from "./navigation/reroute.js";
+export {
+  addErrorHandler,
+  removeErrorHandler,
+} from "./applications/app-errors.js";
+export { mountRootParcel } from "./parcels/mount-parcel.js";
+
+export {
+  NOT_LOADED,
+  LOADING_SOURCE_CODE,
+  NOT_BOOTSTRAPPED,
+  BOOTSTRAPPING,
+  NOT_MOUNTED,
+  MOUNTING,
+  UPDATING,
+  LOAD_ERROR,
+  MOUNTED,
+  UNLOADING,
+  UNMOUNTING,
+  SKIP_BECAUSE_BROKEN,
+} from "./applications/app.helpers.js";
+
+import devtools from "./devtools/devtools";
+import { isInBrowser } from "./utils/runtime-environment.js";
+
+if (isInBrowser && window.__SINGLE_SPA_DEVTOOLS__) {
+  window.__SINGLE_SPA_DEVTOOLS__.exposedMethods = devtools;
+}
+
+declare global {
```

Comment:
I initially had this in a separate file `globals.ts` but ended up colocating it with where `__SINGLE_SPA_DEVTOOLS__` is defined. I have my own typescript patterns that are fairly loose but don't involve a strict folder/file structure. I prefer colocating interfaces in the files that use them much of the time. Perhaps over time, we can develop more established typescript patterns for single-spa projects. I'm open to feedback on typescript patterns - generally I don't prefer lots of folders and files when inlining interfaces does fine